submissions/Levi123/arcade-game/app.js

```diff
@@ -0,0 +1,116 @@
+// Enemies our player must avoid
+const DEFAULT_PLAYER = {
+    sprite: 'images/char-boy.png',
+    positionX: 200,
+    positionY: 400,
+}
+
+const DEFAULT_ENEMY = {
+    sprite: 'images/enemy-bug.png',
+    minSpeed: 200,
+    maxSpeed: 400,
+    widthEnemy: 80,
+    heightEnemy: 60,
+    positionX: -100,
+    positionY: [50, 140, 225],
+}
+
+const DEFAULT_MAP = {
+    widthMap: 505,
+    widthBlock: 101,
+    heightBlock: 82,
+    waterLine: 0,
+}
+
+class Player {
+    constructor(x = DEFAULT_PLAYER.positionX, y = DEFAULT_PLAYER.positionY){
+        this.sprite = DEFAULT_PLAYER.sprite;
+        this.x = x;
+        this.y = y;
+    }
+
+    update(){
+        if (this.y < DEFAULT_MAP.waterLine){
+            setTimeout(() => {
+                this.x = DEFAULT_PLAYER.positionX;
+                this.y = DEFAULT_PLAYER.positionY;
+            }, 300)
```

Comment:
What is 300 semantically? I know it is delay. It delays what?

Reply:
> What is 300 semantically? I know it is delay. It delays what?

when person move to waterline, we refresh him position to default. I added delay to smoother animation.

Reply:
Let the constant name say this somehow